fastlane/screenshots.json

```diff
@@ -63,39 +63,39 @@
 			"filename": "iPhone 11 Pro Max-01.png",
 			"background": "#674399",
 			"screenshot": "iPhone 11 Pro Max-1-dark-order-dashboard.png",
-			"text": "metadata/{locale}/promo_screenshot_1.txt"
+			"text": "metadata/{locale}/app_store_screenshot_1.txt"
```

Comment:
I reverted the commit 8cac92b4cec20c1609b9498ad42d612ff2085c3a that was added in the PR https://github.com/woocommerce/woocommerce-ios/pull/14492, there was just a misunderstanding I think, as the script and config all expects `app_store_screenshot_%`, the cause of the misunderstanding is that for English there was an issue, and now it should be fixed by the change mentioned in the comment ☝️